hm_3/hm_03.py

```diff
@@ -0,0 +1,26 @@
+# Task 1
+string = "I love Python" * 42
+print(string)
+# Task 2
+age_in_month = 24 * 12 + 10
+print(age_in_month)
+# Task 3
+age_in_years = age_in_month // 12
+print(age_in_years)
+# Task 4
+name = "Lolita"
+my_age = f"My name is {name}. I'm {age_in_years} years old"
+print(my_age)
+# Task 5
+i = 1
+print(2 != i)  # True
+print(4 == i)  # False
+print(1 >= i)  # True
+print(5 < i)   # False
+print(8 > i)   # True
+# Task 6
+a = 2
+b = 5
+c = 6
+d = str(a ** ((a + b) + (c - b)))
```

Comment:
You understand me wrong. These variables should be transformed to strings separately: `str(a) + str(b) + str(c)`